Kickstarter-iOS/ViewModels/HelpWebViewModel.swift

```diff
@@ -45,7 +45,7 @@ internal final class HelpWebViewModel: HelpWebViewModelType, HelpWebViewModelInp
   }
 }
 
-private func urlForHelpType(_ helpType: HelpType, baseUrl: URL) -> URL? {
+public func urlForHelpType(_ helpType: HelpType, baseUrl: URL) -> URL? {
```

Comment:
We've made this public so that we can share the functionality, may be worth moving to `SharedFunctions.swift` and adding tests. But I'd suggest doing that in a separate PR.